.github/workflows/compliance.yml

```diff
@@ -0,0 +1,49 @@
+name: Run compliance tests
+
+on:
+  workflow_dispatch:
+    inputs:
+#      test_repo_url:
+#        description: Tests repository (eg. owner/repo)
+#        required: true
+#        default: elixir-cloud-aai/tes-compliance-suite
+      server:
+        description: The API server URL
+        required: true
```

Comment:
As I said before, having tests that rely on a compliant server is a sort of circular test. It will fail if and _only if_ tests are okay _and_ the server is compliant. However, the latter we only know if the tests pass...

So I really think we should manually check whether tests work and make sense. If not, sooner or later we would get feedback from implementers who are _sure_ that their implementation is correct. We can then look at the tests and see what to do.

So in essence, I think we should _not_ run tests against a particular implementation/instance in the CI. Rather, we should only check the validity of the tests.

Reply:
If not already available, the test suite should have a mode that validates the tests only (that should not require a server)